core/storage/skill/gae_models.py

```diff
@@ -138,18 +138,10 @@ def _trusted_commit(
             committer_user_settings_model.username
             if committer_user_settings_model else '')
 
-        skill_rights = SkillRightsModel.get_by_id(self.id)
-
-        status = ''
-        if skill_rights.skill_is_private:
-            status = constants.ACTIVITY_STATUS_PRIVATE
-        else:
-            status = constants.ACTIVITY_STATUS_PUBLIC
-
         skill_commit_log_entry = SkillCommitLogEntryModel.create(
             self.id, self.version, committer_id, committer_username,
             commit_type, commit_message, commit_cmds,
-            status, False
+            constants.ACTIVITY_STATUS_PUBLIC, False
```

Comment:
is there any specific reason to have this value as there isn't any public/private terminology for skills?

Reply:
This is a required field for the `BaseCommitLogEntryModel`, right? That's why I set it to public. Besides, no split between terminology means that all of them are public, so this fits.

Reply:
I see, Thanks for the explanation.